### DISCUSSION TITLE
misurazione energia trifase

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#3120] [HUB: Home Assistant]
vorrei implementare nel mio HA la misurazione della corrente consumate sia nella totalità che nelle varie zone.
avrei pensato allo Shelly EM che mi risolve sicuramente la misurazione delle singole zone. 
mi rimane il problema per il controllo generale perché ho un contatore trifase.
grazie

**Comment by Scialla** [comment]
Shelly 3EM
